notebooks/sen2_blade_csv_plans_placements.clj

```diff
@@ -36,21 +37,21 @@
 
 ;;; ## Parameters
 ;;; ### Working directory
-;; For output files:
-^{::clerk/viewer clerk/md}
+^{::clerk/visibility {:code :show, :result :hide}
```

Comment:
worth adding a section above saying what is taken in (blade export files?) and what is produced at the end? (basic csv extract, issues file, updated csv extract after modification file applied?)

I think I get a bit confused as the notebook sort of needs to be run in two bits. Once to do the extract, then you look at it, and then the second part to do the updates to the raw extract. Do I have that correctly?

Reply:
I've added an explanatory section at the top of the notebook and described how in practice the "update" step would be done elsewhere.